src/main/java/lotto/model/PriceResult.java

```diff
@@ -0,0 +1,34 @@
+package lotto.model;
+
+import java.util.HashMap;
+
+public class PriceResult {
+    private final HashMap<Price, Integer> result;
+
+    public PriceResult() {
+        result = new HashMap<>();
+        for (Price price : Price.values()) {
+            if (price!= Price.NOTHING) {
+                result.put(price, 0);
+            }
+        }
+    }
+
+    public HashMap<Price, Integer> getResult() {
+        return result;
+    }
```

Comment:
```suggestion
    public Map<Price, Integer> getResult() {
        return result;
    }
```

구현 클래스보다는 interface로 구현하면 추후 구현체가 HashMap이 아닌 TreeMap과 같은 다른 구현체로 변경해도 이 메서드와 의존하고 있는 다른 코드는 변경이 발생하지 않을 수도 있지 않을까?